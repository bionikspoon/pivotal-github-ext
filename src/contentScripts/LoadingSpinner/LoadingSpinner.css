.LoadingSpinner__container {
  margin-left: -60px;

  & > svg {
    animation: rotator 1.4s cubic-bezier(0.68, -0.55, 0.27, 1.55) infinite;
  }
}
@keyframes rotator {
  0% {
    transform: rotate(0deg);
  }
  100% {
    transform: rotate(360deg);
  }
}
