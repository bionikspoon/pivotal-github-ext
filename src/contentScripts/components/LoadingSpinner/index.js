/* @flow */
export { default } from './LoadingSpinner'
