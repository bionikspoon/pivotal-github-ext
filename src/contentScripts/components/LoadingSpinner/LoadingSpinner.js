/* @flow */
import React from 'react'
import PivotalLogo from '../PivotalLogo'
import './LoadingSpinner.css'

export default function LoadingSpinner() {
  return <PivotalLogo className="LoadingSpinner__container" />
}

LoadingSpinner.propTypes = {}
