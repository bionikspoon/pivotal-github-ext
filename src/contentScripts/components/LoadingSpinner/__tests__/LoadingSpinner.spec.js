import React from 'react'
import { shallow } from 'enzyme'
import LoadingSpinner from '../LoadingSpinner'

describe('components/LoadingSpinner', () => {
  test('it renders correctly', () => {
    const props = {}
    const wrapper = shallow(<LoadingSpinner {...props} />)

    expect(wrapper).toMatchSnapshot()
  })
})
