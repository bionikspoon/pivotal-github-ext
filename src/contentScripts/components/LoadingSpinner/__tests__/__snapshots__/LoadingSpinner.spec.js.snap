// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/LoadingSpinner it renders correctly 1`] = `
<PivotalLogo
  className="LoadingSpinner__container"
  fill="#ffffff"
/>
`;
